samples/operate-first/Makefile

```diff
@@ -0,0 +1,20 @@
+ROOT_DIR := ../..
+include $(ROOT_DIR)/Makefile.env
+export RELEASE_NAME=m4d
+export NAMESPACE=m4d-system
+export M4D_CHART_PATH=$(ROOT_DIR)/charts/m4d
+export CRD_CHART_PATH=$(ROOT_DIR)/charts/m4d-crd
+
+.PHONY: helm-template
+helm-template: $(TOOLBIN)/yq $(TOOLBIN)/helm
+	$(TOOLBIN)/helm template $(RELEASE_NAME) -n $(NAMESPACE) $(M4D_CHART_PATH) | $(TOOLBIN)/yq e 'select(.kind=="Namespace" or .kind=="MutatingWebhookConfiguration" or .kind=="ValidatingWebhookConfiguration" or .kind=="ClusterRole" or .kind=="ClusterRoleBinding")' - > m4d.yaml
+	$(TOOLBIN)/helm template $(RELEASE_NAME) -n $(NAMESPACE) $(CRD_CHART_PATH) > m4d-crd.yaml
+	python -m pip install -r requirements.txt
+	python filterClusterScoped.py
+
+.PHONY: generate
+generate:
+	python -m pip install -r requirements.txt
+	python filterClusterScoped.py
+
+all: helm-crd-hook helm-verify
```

Comment:
1. Change `all` to be the first target in this Makefile (i.e. move it to before the `helm-template` target) so it would be the default target
2. You probably meant `all: helm-template generate` because `helm-crd-hook` and `helm-verify` do not exist here
3. add `.PHONY: all` like in the other targets